changelog.d/11659.bugfix

```diff
@@ -0,0 +1 @@
+Include the bundled aggregations in the `/sync` response.
```

Comment:
Is this part of an MSC by any chance? If so maybe we should link it here?

Reply:
It is part of MSC2675, yes.

See https://github.com/uhoreg/matrix-doc/blob/aggregations-helpers/proposals/2675-aggregations-server.md#bundling

Reply:
```suggestion
Include the bundled aggregations in the `/sync` response, per [MSC2675](https://github.com/matrix-org/matrix-doc/pull/2675).
```